Src/Support/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -314,11 +314,12 @@ public void Download_Error_JsonResponse()
         }
 
         [Test]
-        public void Download_Error_PlaintextResponse()
+        [TestCase("Not Found")]
+        [TestCase("")]
+        public void Download_Error_PlaintextResponse(string responseText)
         {
             var downloadUri = "http://www.sample.com";
             var chunkSize = 100;
-            var responseText = "Not Found";
 
             var handler = new MultipleChunksMessageHandler { ErrorResponse = responseText };

```

Comment:
Yes, parsedResponse will be null when responseText is "", because `service.Serializer.Deserializer<StandardResponse<object>>("")` will return null.
See https://github.com/google/google-api-dotnet-client/blob/1dce119eb5200be57a6ff24159b895594d6bd558/Src/Support/GoogleApis.Core/Apis/Json/NewtonsoftJsonSerializer.cs#L114